### ISSUE TITLE
Bulma.select properties is generated on the wrong HTML element

### ISSUE THREAD

**Issue by samme78** [Dzoukr/Feliz.Bulma#10]
Using properties on a `Bulma.select` element does not work due to the class name ending up on the inner `<select>` tag rather than the surronding `<div>`

Example:

```
Bulma.field [
     Bulma.select [
         select.isDanger ] ]
```

generates the following HTML

```
<div class="field">
    <div class="select">
        <select class="is-danger"></select>
    </div>
</div>
```

The dropdown is not styled correctly.

For the styling to work the following HTML should be generated

```
<div class="field">
    <div class="select is-danger">
        <select></select>
    </div>
</div>
```

**Comment by Dzoukr** [comment]
Hi @samme78,

looking at documentation, you are right. I'll look at it soon, probably need to change how whole `select` is rendered (wrap all in div by default).

**Comment by Dzoukr** [comment]
Fixed in `1.9.1`
